drivers/modem/modem_pin.c

```diff
@@ -19,19 +19,13 @@
 int modem_pin_read(struct modem_context *ctx, u32_t pin)
 {
 	int ret = 0;
```

Comment:
I now get a build warning since `ret` is no longer used. Can this be removed? Here's my build output:

```
zephyrproject/zephyr/drivers/modem/modem_pin.c: In function 'modem_pin_read':
zephyrproject/zephyr/drivers/modem/modem_pin.c:21:6: warning: unused variable 'ret' [-Wunused-variable]
  int ret = 0;
      ^~~
```

Reply:
@brooksprumo thanks for testing. I've fixed this now.